products/zoningtaxlots/models/intermediate/int__commercialoverlay.sql

```diff
@@ -47,10 +47,43 @@ commoverlayperorder AS (
     FROM commoverlayper
 ),
 
-filter AS (
-    SELECT *
-    FROM commoverlayperorder
+filtered AS (
+    SELECT * FROM commoverlayperorder
     WHERE perbblgeom >= 10 OR perzonegeom >= 50
```

Comment:
I tried doing those in one CTE. Since I can't refer to the alias specified in the SELECT clause in WHERE clause, I did `WHERE (segbblgeom / allbblgeom) * 100 >= 10 OR (segzonegeom / allzonegeom) * 100 >=  50`. I am unsure if it was because of the rounding or differences in SQL calculation, but this produced different results from referring to aliases in a separate query (one row was different).

Reply:
Let's include it, and look at the difference. I wouldn't be shocked if this is an edge case where we'd be correcting something.

Reply:
@fvankrieken This results in 2 different rows according to `bbldiff`